client/scss/settings/_variables.scss

```diff
@@ -1,19 +1,11 @@
 @use 'sass:color';
 // paths
-
 // We can't use absolute paths here, because those are dependent on Django's
 // STATIC_URL setting. Instead, relative paths from the final location of the
 // compiled CSS files are used.
 // */
-
 $images-root: '../images/';
 
-// grid settings
-$grid-columns: 12;
-$grid-gutter-width: 3%;
-$grid-max-width: 1200px;
-$grid-content-indent: 0.7;
-
 $mobile-nice-padding: 20px;
 $desktop-nice-padding: 80px;
```

Comment:
Might be nice to update these two variables to their linked them sizing while we are here.


```
$mobile-nice-padding: theme('spacing.5');
$desktop-nice-padding: theme('spacing.20');
```